caffe2/operators/experimental/c10/cpu/stop_gradient_cpu.cc

```diff
@@ -18,11 +18,23 @@ void stop_gradient_op_cpu_impl(
     output.CopyFrom(input);
   }
 }
+
+static auto registry = c10::RegisterOperators().op(
+    FunctionSchema(
+        "_c10_experimental::StopGradient",
+        "",
```

Comment:
Empty overload string still looks really bad lol

Reply:
This will look much better once we use the string based schema.